stories/packages/LocationSelectorUtils.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { initGoogleMapServices, getAddressComponents, Input, Button } from '@textkernel/oneui';
+import { ensureApiKey } from '../utils/ensureApiKey';
+
+const columnStyle = {
+    width: '30%',
+    padding: '1em',
+};
+
+storiesOf('packages|LocationSelectorUtils', module).add('LocationSelectorUtils', () => {
```

Comment:
"LocationSelectorUtils" sounds too common, as I see we demonstrate here only `getAddressComponents` util

Reply:
A good point, I will change:
```
storiesOf('packages|LocationSelectorUtils', module).add('getAddressComponents',
```